src/activity-logger/background/log-page-visit.test.js

```diff
@@ -0,0 +1,40 @@
+/* eslint-env jest */
+/* eslint import/namespace: "off" */
+
+import maybeLogPageVisit from 'src/activity-logger/background/log-page-visit'
+import * as storePage from 'src/page-storage/store-page'
+import db from 'src/pouchdb'
+
+afterAll(async () => {
+    await db.destroy()
+})
+
+describe('maybeLogPageVisit', () => {
```

Comment:
@Treora should I wait for the changes to be pushed and then proceed to rebase and test, or would you like me to refactor the code with the tests and modify the pull request?